backend/notifications.js

```diff
@@ -0,0 +1,44 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+
+const DaysRange = 3;
+const conditionWeights = {
```

Comment:
so if i'm understanding this correctly, the age of a patient is more important in deciding if notifications can be turned on/off than the appointment number?

Reply:
Yes! 1 represents least important and 4 represents most important.